x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/analytics_router.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+
+import { APP_SEARCH_PLUGIN } from '../../../../../common/constants';
+import { SetAppSearchChrome as SetPageChrome } from '../../../shared/kibana_chrome';
+import { NotFound } from '../../../shared/not_found';
+import {
+  ENGINE_PATH,
+  ENGINE_ANALYTICS_PATH,
+  ENGINE_ANALYTICS_TOP_QUERIES_PATH,
+  ENGINE_ANALYTICS_TOP_QUERIES_NO_RESULTS_PATH,
+  ENGINE_ANALYTICS_TOP_QUERIES_NO_CLICKS_PATH,
+  ENGINE_ANALYTICS_TOP_QUERIES_WITH_CLICKS_PATH,
+  ENGINE_ANALYTICS_RECENT_QUERIES_PATH,
+  ENGINE_ANALYTICS_QUERY_DETAIL_PATH,
+} from '../../routes';
+import {
+  ANALYTICS_TITLE,
+  TOP_QUERIES,
+  TOP_QUERIES_NO_RESULTS,
+  TOP_QUERIES_NO_CLICKS,
+  TOP_QUERIES_WITH_CLICKS,
+  RECENT_QUERIES,
+} from './constants';
+
+interface Props {
+  engineBreadcrumb: string[];
+}
+export const AnalyticsRouter: React.FC<Props> = ({ engineBreadcrumb }) => {
+  const ANALYTICS_BREADCRUMB = [...engineBreadcrumb, ANALYTICS_TITLE];
+
+  return (
+    <Switch>
+      <Route exact path={ENGINE_PATH + ENGINE_ANALYTICS_PATH}>
+        <SetPageChrome trail={ANALYTICS_BREADCRUMB} />
+        TODO: Analytics overview
```

Comment:
Something I was thinking about recently was a <Todo> component that would hide itself outside of dev mode.

```
<Todo>
Todo: To queries with clicks
</Todo>
```

Could save us some trouble down the line if we're in the habit of adding these to the markup

Reply:
I'm not bothered about it - it's a massive migration in progress, it's going to have TODO's. It's not something I anticipate having once the migration is complete. Additionally all of master is essentially dev mode & and none of these views are visible in 7.x releases, so it's not concerning in that regard.